### PR TITLE
Cast contentId and locationId command arguments to integer

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 4.6, master
| Edition       | All

Fix `PHP Fatal error:  Uncaught TypeError: Ibexa\Core\Repository\SiteAccessAware\LocationService::loadLocation(): Argument #1 ($locationId) must be of type int, string given, called in /var/www/html/src/Command/BrowseLocationsCommand.php on line 45 and defined in /var/www/html/vendor/ibexa/core/src/lib/Repository/SiteAccessAware/LocationService.php:53`

I didn't test the others but casting every 'contentId' or 'locationId' to `int` is probably a good habit.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ private function browseLocation(Location $location, OutputInterface $output, int
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $locationId = $input->getArgument('locationId');
+        $locationId = (int) $input->getArgument('locationId');
```
Comment (adriendupuis) [inline code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php:43]: @mnocon Why PHPStan didn't see an issue here?

https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Console/Input/InputInterface.php#L83-L90 declares returning `mixed` but in fact it's probably only `int|array`.

Comment (mnocon) [reply code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php:43]: This one would be detected using level 9:
https://phpstan.org/user-guide/rule-levels

Bumping it results in:
```
  Line   api/public_php_api/src/Command/BrowseLocationsCommand.php                                                                     
 ------ ------------------------------------------------------------------------------------------------------------------------------ 
  :45    Parameter #1 $locationId of method Ibexa\Contracts\Core\Repository\LocationService::loadLocation() expects int, mixed given.  
 ------ ------------------------------------------------------------------------------------------------------------------------------ 
```

(and another 147 errors)

Comment (mnocon) [reply code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php:43]: We can think if we want to introduce this (level 10 is the highest right now), in product code we use level 8

**Review by mnocon — APPROVED** [review]

File: code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php (hunk 1 above)
Comment (adriendupuis) [reply code_samples/api/public_php_api/src/Command/BrowseLocationsCommand.php:43]: @mnocon I don't fix 147 errors here 😛. I would say that we don't need it in CI. I'm curious about those level 9 new errors. But there is already a hundred to check in the actual level 8 phpstan-baseline.neon 😅

**Pull request merged**